src/openapi/transport/core.spec.js

```diff
@@ -865,4 +865,32 @@ describe('openapi TransportCore', () => {
             fetch.mockClear();
         });
     });
+
+    describe('abort signal', () => {

```

Comment:
Nitpick. Let's fix prettier errors.

![image](https://user-images.githubusercontent.com/117785/118111179-a14c1c80-b3e3-11eb-8703-69834a5abb59.png)

Reply:
Resolved it as part of preparing beta build/